libsolidity/codegen/ArrayUtils.cpp

```diff
@@ -335,9 +335,14 @@ void ArrayUtils::copyArrayToMemory(ArrayType const& _sourceType, bool _padToWord
 		if (baseSize > 1)
```

Comment:
8 lines above (line `327`): the comment `memcpy using the build-in contract` is no longer always valid.